Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -838,11 +838,11 @@ def search_indicator(indicator_type, indicator_value):
             text_error = {}
         error_message = text_error.get('message')
         if error_message:
-            return_error(f'Request Failed with status: {result.status_code}.\n'
-                         f'Reason is: {str(error_message)}.')
+            raise ValueError(f'Request Failed with status: {result.status_code}.\n'
+                             f'Reason is: {str(error_message)}.')
         elif str(result.status_code) in ERROR_DICT:
-            return_error(f'Request Failed with status: {result.status_code}.\n'
-                         f'Reason is: {ERROR_DICT[str(result.status_code)]}.')
+            raise ValueError(f'Request Failed with status: {result.status_code}.\n'
+                             f'Reason is: {ERROR_DICT[str(result.status_code)]}.')
         else:
```

Comment:
why did you change it?

Reply:
when it was return_error it skipped the 'except' I entered in the search_file_command function bellow.
This is a way to use it and not change the integration behavior.